src/lightning/fabric/strategies/xla.py

```diff
@@ -82,6 +82,22 @@ def checkpoint_io(self) -> CheckpointIO:
     def checkpoint_io(self, io: Optional[CheckpointIO]) -> None:
         self._checkpoint_io = io
 
+    @property
+    def global_rank(self) -> int:
+        return super().global_rank if self._launched else 0
+
+    @property
+    def local_rank(self) -> int:
+        return super().local_rank if self._launched else 0
+
+    @property
+    def node_rank(self) -> int:
+        return super().node_rank if self._launched else 0
+
+    @property
+    def world_size(self) -> int:
+        return super().world_size if self._launched else 1
```

Comment:
can we add a test explicitly covering the super part? Sometimes python is funny with calling properties from
base classes...

Reply:
Good suggestion. Done!